lib/automation/report_runner.rb

```diff
@@ -113,19 +121,22 @@ def run!
           result << response
         end
 
-        post =
-          PostCreator.create!(
-            @sender,
-            raw: result,
-            title: @title,
-            archetype: Archetype.private_message,
-            target_usernames: @receivers.map(&:username).join(","),
-            skip_validations: true,
-          )
-
-        if @debug_mode
-          input = input.split("\n").map { |line| "    #{line}" }.join("\n")
-          raw = <<~RAW
+        users = @receivers.map(&:username).join(",")
```

Comment:
nit, but I think this should be called `receiver_usernames` or something, otherwise this variable could appear to be an array of User